src/kili/graphql/operations/api_key/queries.py

```diff
@@ -0,0 +1,59 @@
+"""
+GraphQL Queries of API keys
+"""
+
+
+from typing import Optional
+
+from kili.graphql import BaseQueryWhere, GraphQLQuery
+
+from ....types import ApiKey
+
+
+class APIKeyWhere(BaseQueryWhere):
+    """
+    Tuple to be passed to the APIKeyQuery to restrict the query
+    """
+
+    def __init__(
+        self,
+        api_key_id: Optional[str] = None,
+        user_id: Optional[str] = None,
+        api_key: Optional[str] = None,
+    ):
+        self.api_key_id = api_key_id
```

Comment:
you probably should add the type, I am not sure it is propagated.

Reply:
`self.api_key_id: Optional[str] = api_key_id`

Reply:
by hovering on the variable in the functions below it shows the type.
If it appears propagated, I prefer not to add it not to overload the code